packages/govern-server/src/core/data/index.ts

```diff
@@ -54,78 +53,90 @@ class ConnectorTheGraph {
     })
   }
 
-  private async fetchResult<T>(
+  private async fetchResult<ReturnType>(
```

Comment:
side-note: Is it a standard here at Aragon to not name the file as the class within is named?

Reply:
We usually do it except in certain cases, and I think we should do it here :smiley:

Reply:
Done in https://github.com/aragon/govern/pull/60/commits/0ed5daa48b442e74106fb9234c0a7652fd9e19e3 and https://github.com/aragon/govern/pull/60/commits/41c5103c92194ffe92670270173cc8f87ffc5a04